include/unifex/scheduler_concepts.hpp

```diff
@@ -199,6 +205,34 @@ UNIFEX_CONCEPT //
     UNIFEX_FRAGMENT(unifex::_scheduler_provider, SP);
 #endif
 
+namespace _schedule
+{
+  struct _fn {
+  private:
+    template <typename Scheduler>
+    static auto impl_(Scheduler sched)
+        noexcept(noexcept(make_sender_for<schedule>(
+            _impl{}((Scheduler&&) sched), get_scheduler = Scheduler(sched)))) {
```

Comment:
Yes, it may be too clever. I am indeed using this as a kind of named parameter, except it's like a little DSL for permitting a caller to say, I would like the resulting type to customize this particular CPO and here is the corresponding value. Suggestions welcome.